presto-docs/src/main/sphinx/release/release-0.238.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.238
+=============
+
+**Highlights**
+==============
+* Fix SQL function parameters to be case-insensitive.
+* Add support to create external function. (:issue:`13254`)
+* Add support for exchange materialization of table bucketed by non-hive types. This can be enabled by setting the ``bucket_function_type_for_exchange`` session property or ``hive.bucket-function-type-for-exchange `` configuration property to ``PRESTO_NATIVE``
+* Add predicate pushdown support for ``DATE``, ``TIMESTAMP``, and ``TIMESTAMP_WITH_TIME_ZONE`` literals for Pinot connector.
+
+General Changes
+_______________
+* Fix SQL function parameters to be case-insensitive.
+* Add ``target_result_size`` session property to customize data batch sizes being streamed from coordinator.
+* Add optimization to push null filters to the INNER side of equijoins. The optimization can be enabled with ``optimize-nulls-in-joins``.
+* Add ``sessionProperty`` override to Presto JDBC URI.
+* Add support to create external function. (:issue:`13254`)
+* Remove max buffer count configuration property ``driver.max-page-partitioning-buffer-count`` for optimized repartitioning.
+
+Geospatial Changes
+__________________
+* Fix error in :func:geometry_invalid_reason.
```

Comment:
So github nicely formatted my comment - please see https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#formatting for how to format function names